### DISCUSSION TITLE
Bias in Species Recognition Favoring American Species Over Eurasian Ones

### DISCUSSION THREAD

**Discussion by tphakala** [birdnet-team/BirdNET-Analyzer#424] [General]
I am seeing detection bias to american members of certain families, for example eurasian treecreeper is most of time detected as brown creeper but because of range filter this detection is ignored. I guess I can locally create aliases by tweaking labels but I guess there is some improvements in detection model also.

**Comment by Mattk70** [comment]
With the 'range' filter, the top detections are suppressed if they are not likely in your area.  The most likely detection that *is* in your area should be shown. Are, in this example, the Eurasion Treecreepers not appearing?

**Comment by tphakala** [comment]
How is this completed?

**Comment by tphakala** [comment]
Range filter option is not a proper fix for this, eurasian treecreeper is still poorly detected. However I have solved this issue by training my own classifier with local bird calls.
